src/pages/home/ConnectWalletOptions.vue

```diff
@@ -0,0 +1,163 @@
+
+
+<script lang="ts">
+import { computed, defineComponent, inject, ref, watch } from 'vue';
+import { useAccountStore } from 'src/antelope/stores/account';
+import { useChainStore } from 'src/antelope/stores/chain';
+import { Web3Modal } from '@web3modal/html';
+import { EthereumClient } from '@web3modal/ethereum';
+import { getAccount } from '@wagmi/core';
+import { useEVMStore, usePlatformStore } from 'src/antelope';
+
+export default defineComponent({
+    name: 'ConnectWalletOptions',
+    props: {
+        toggleWalletConnect: {
+            required: true,
+            type: Boolean,
+        },
+    },
+    setup(props){
+        const wagmiClient = inject('$wagmi') as EthereumClient;
+        const web3Modal = ref<Web3Modal>();
+        const supportsMetamask = computed(() => useEVMStore().isMetamaskSupported);
```

Comment:
This path solves the problem [found here](https://github.com/telosnetwork/telos-wallet/pull/239#issuecomment-1509438649)

```suggestion
        const supportsMetamask = computed(() =>
            useEVMStore().isMetamaskSupported || localStorage.getItem('wagmi.connected'));
```

Reply:
resolved